manager/aica_django/converters/Knowledge.py

```diff
@@ -757,9 +774,19 @@ def nginx_accesslog_to_knowledge(
         )
 
     my_hostname = socket.gethostname()
-    my_ipv4 = IPv4Address(socket.gethostbyname(my_hostname))
-    my_ipv4_knowledge = my_ipv4.to_knowledge_node()
-    knowledge_nodes.append(my_ipv4_knowledge)
+    try:
```

Comment:
What happens if both IPv4 and IPv6 exceptions are hit?

Reply:
That means the localhost does not have an IP4 or IP6 itself. In this case it would not reach this far because it will not be able to bind a port and start the app itself.
Next to this nothing happens because these knowledgenodes are not used later (with relations). A later improvement would be to add the relation between hostname and ip.